data/gamemoderun.in

```diff
@@ -1,11 +1,44 @@
 #!/bin/bash
 # Helper script to launch games with gamemode
 
+# Load vsync mode via dbus
+vsync_mode="$(dbus-send --session --dest=com.feralinteractive.GameMode --type=method_call --print-reply /com/feralinteractive/GameMode com.feralinteractive.GameMode.GetVsyncMode | grep -oP '(?<=").*(?=")')"
+
+# Load hybrid GPU mode via dbus
+hybrid_gpu_mode="$(dbus-send --session --dest=com.feralinteractive.GameMode --type=method_call --print-reply /com/feralinteractive/GameMode com.feralinteractive.GameMode.GetHybridGPUMode | grep -oP '(?<=").*(?=")')"
+
+# Set vsync mode
+if [ "$vsync_mode" == "force_disable" ]; then
+  echo "gamemoderun: force disable vsync"
+  export vblank_mode=0
+elif [ "$vsync_mode" == "force_enable" ]; then
+  echo "gamemoderun: force enable vsync"
+  export vblank_mode=1
+fi
+
+# Set hybrid GPU mode
+if [ "$hybrid_gpu_mode" == "prime" ]; then
+  echo "gamemoderun: launch with prime"
+  exec_cmd="exec"
+  export DRI_PRIME=1
+elif [ "$hybrid_gpu_mode" == "bumblebee" ]; then
+  echo "gamemoderun: launch with bumblebee"
```

Comment:
Given exec_cmd usually matches the GPU mode config, why not have these two match as well? It'd remove a big chunk of complexity

Reply:
I think thats a good idea. I used the method to avoid Errors if the D-BUS won't work, but if we use a call in `main.c`, we can simply output the complete `exec_cmd` and get rid of all the code.